cpp/src/glm/ols.cuh

```diff
@@ -61,9 +61,9 @@ void olsFit(const raft::handle_t &handle, math_t *input, int n_rows, int n_cols,
   ASSERT(n_cols > 0, "olsFit: number of columns cannot be less than one");
```

Comment:
Is the local variable "allocator" no longer needed in all these examples? Would be nice to delete if so.

Reply:
Oh, nevermind, I think I see it is passed later to lstsqQR